src/AutoRest.CSharp/Common/Output/Models/Types/SchemaObjectType.cs

```diff
@@ -107,7 +92,7 @@ protected SchemaObjectType(ObjectSchema objectSchema, string defaultNamespace, T
                 var existingMember = ModelTypeMapping?.GetMemberByOriginalName("$AdditionalProperties");
 
                 _additionalPropertiesProperty = new ObjectTypeProperty(
-                    BuilderHelpers.CreateMemberDeclaration("AdditionalProperties", ImplementsDictionaryType, "public", existingMember, _typeFactory),
+                    BuilderHelpers.CreateMemberDeclaration("AdditionalProperties", ImplementsDictionaryType, "public", existingMember, MgmtContext.TypeFactory),
```

Comment:
so we are changing this type to be "MPG only type provider"? But I still see we have a `_typeFactory` field in this class, we might need to align

Reply:
was in an intermediate stage to have `MgmtContext`, updated to use `_typeFactory` instead